spa/src/pages/StartConnection/index.tsx

```diff
@@ -128,14 +131,17 @@ const StartConnection = () => {
 				>
 					Continue
 				</Button>
-				<Button
-					appearance="subtle"
-					onClick={() => {
-						navigate("connections");
-					}}
-				>
-					Go to backfill page
-				</Button>
+				{
+					enable5KuExperienceBackfillPage &&
+						<Button
+							appearance="subtle"
+							onClick={() => {
+								navigate("/spa/connections");
+							}}
+						>
+							Go to backfill page
+						</Button>
+				}
```

Comment:
Dont have to do anything, just a Q.
Im very late to the spa party, but do we want to be including the accessibility attributes e.g. aria-label